src/o365/spo/commands/file/file-get.ts

```diff
@@ -0,0 +1,277 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import { ContextInfo } from '../../spo';
+import Utils from '../../../../Utils';
+import Auth from '../../../../Auth';
+import fs = require('fs');
```

Comment:
For loading `fs` we can use the same construct as for loading `path`

Reply:
Changed accordingly